### ISSUE TITLE
Dependencies don't get updated if the version is delegated

### ISSUE THREAD

**Issue by cwrau** [patrikerdes/gradle-use-latest-versions-plugin#14]
I have the following `build.gradle.kts`:

``` kotlin
val junitVersion: String by project

dependencies {
	testRuntimeOnly("org.junit.jupiter", "junit-jupiter-engine", junitVersion)
}
```
and my `gradle.properties`:

``` properties
junitVersion=5.3.1
```

If I run `./gradlew dependencyUpdates` I can see that there is an update for `junit` to `5.3.2`.

But if I run `./gradlew useLatestVersions` the `gradle.properties` does not get updated.

**Comment by usr42** [comment]
Which version of the plugin do you use?

**Comment by cwrau** [comment]
I'm using the following versions:

``` kotlin
     id("se.patrikerdes.use-latest-versions") version "0.2.7"
     id("com.github.ben-manes.versions") version "0.20.0"
```

**Comment by usr42** [comment]
@cwrau I can reproduce the issue, but it is not related to the delegete.
It happens because you use `testRuntimeOnly` not with the dependencyNotation (`testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$junitVersion")`, but with explicit group, name and version without named parameters.
This doesn't seem to be supported yet.

I will write some more test cases for the different possibilities to specify the dependencies with the Kotlin DSL and will open a PR to add this functionality.

**Comment by patrikerdes** [comment]
@usr42 thanks, all contributions are very welcome

**Comment by usr42** [comment]
@cwrau This should be fixed now

**Comment by cwrau** [comment]
Amazing!
When can I expect a release?
Or is it automagically deployed?

**Comment by usr42** [comment]
It's not automatically deployed, so we have to wait for a release.
@patrikerdes If you need any help, please let me know. Is for example some Readme adaptation needed?

**Comment by patrikerdes** [comment]
It has been released as 0.2.8

**Comment by cwrau** [comment]
Thank you for the quick work!
